test/e2e/scaffold/scaffold.go

```diff
@@ -442,6 +442,8 @@ func (s *Scaffold) beforeEach() {
 func (s *Scaffold) afterEach() {
 	defer ginkgo.GinkgoRecover()
 
+	shouldKeepNamespace := false
+
 	if ginkgo.CurrentSpecReport().Failed() {
```

Comment:
If we keep the namespace, we don't need to dump namespace contents anymore.

Reply:
Is three any situation we still need real-time information from the namespace? Sometimes e2e-test may spend over an hour. Some resources like `Event` might be deleted.

Reply:
We do not currently dump this resource either.